engine/apps/mobile_app/alert_rendering.py

```diff
@@ -9,11 +9,30 @@ def _render_for(self):
         return "MOBILE_APP"
 
 
-def get_push_notification_message(alert_group):
+def get_push_notification_subtitle(alert_group):
     alert = alert_group.alerts.first()
     templated_alert = AlertMobileAppTemplater(alert).render()
-    title = str_or_backup(templated_alert.title, "Alert Group")
+    alert_title = str_or_backup(templated_alert.title, "Alert Group")
 
-    return emojize(
-        f"#{alert_group.inside_organization_number} {title} via {alert_group.channel.short_name}", use_aliases=True
+    status_verbose = "Firing"  # TODO: we should probably de-duplicate this text
+    if alert_group.resolved:
+        status_verbose = alert_group.get_resolve_text()
+    elif alert_group.acknowledged:
+        status_verbose = alert_group.get_acknowledge_text()
+
+    number_of_alerts = alert_group.alerts.count()
+    if number_of_alerts <= 10:
+        alerts_count_str = str(number_of_alerts)
+    else:
+        alert_count_rounded = (number_of_alerts // 10) * 10
+        alerts_count_str = f"{alert_count_rounded}+"
+
+    alert_status = f"Status: {status_verbose}, alerts: {alerts_count_str}"
+
+    subtitle = (
+        f"#{alert_group.inside_organization_number} {alert_title}\n"
+        + emojize(f"via {alert_group.channel.short_name}", use_aliases=True)
+        + f"\n{alert_status}"
     )
```

Comment:
you may want to pass the entire string assigned to the `subtitle` variable to the `emojize` function. If `alert_title` contains any emoji codes they won't be properly rendered I believe (may want to test this).

Reply:
I didn't do that just because currently that's how we do it, we don't "emojize" the alert title. 
But actually I think it's just better to do it